libs/components/errors/testing/src/modules/error/error-harness.ts

```diff
@@ -0,0 +1,80 @@
+import { HarnessPredicate, TestElement } from '@angular/cdk/testing';
+import { SkyComponentHarness } from '@skyux/core/testing';
+import { SkyErrorType } from '@skyux/errors';
+
+import { SkyErrorHarnessFilters } from './error-harness-filters';
+
+/**
+ * Harness for interacting with an error component in tests.
+ */
+export class SkyErrorHarness extends SkyComponentHarness {
+  /**
+   * @internal
+   */
+  public static hostSelector = 'sky-error';
+
+  #actionButton = this.locatorForOptional('sky-error-action button');
+  #description = this.locatorFor('.sky-error-description');
+  #imageContainer = this.locatorForOptional('.sky-error-image-container');
+  #title = this.locatorFor('.sky-error-title');
+
+  /**
+   * Gets a `HarnessPredicate` that can be used to search for a
+   * `SkyErrorHarness` that meets certain criteria.
+   */
+  public static with(
+    filters: SkyErrorHarnessFilters,
+  ): HarnessPredicate<SkyErrorHarness> {
+    return SkyErrorHarness.getDataSkyIdPredicate(filters);
+  }
+
+  /**
+   * Clicks the action button in the error action.
+   */
+  public async clickActionButton(): Promise<void> {
+    await (await this.#getAction()).click();
+  }
+
+  /**
+   * Gets the text of the action button in the error action.
+   */
+  public async getActionButtonText(): Promise<string> {
+    return await (await this.#getAction()).text();
+  }
```

Comment:
while `sky-error-action` with a button inside is what we have shown consumers in code examples, this feels like overstepping in what we provide consumers to interact with parts they own.

I think to align with how we have things setup in other harnesses, we should provide consumers a queryable harness for `sky-error-action`. 

However I do like these being provided as helper functions bc even in our docs for `sky-error-action` the only example we give for what they can put inside it is a button.

Reply:
added queryable container for error action